app/components/notification-dropdown.js

```diff
@@ -3,4 +3,34 @@ import Ember from 'ember';
 const { Component } = Ember;
 
 export default Component.extend({
+  didInsertElement() {
+    this._super.call(this);
+    this.$('.mail.outline.icon').popup({
```

Comment:
Popup shouldn't be initialised on the icon. It should be initialised on the component itself.

use `this.$()` instead of `this.$('.mail.outline.icon')`. And also, be be sure to destroy the pop up in `willDestroyElement`